client/components/event-form.jsx

```diff
@@ -80,19 +101,22 @@ export default function EventForm() {
         headers: {
           'Content-Type': 'application/json'
         },
-        body: JSON.stringify({ title, description, timestamp, origin, destination, coords })
+        body: JSON.stringify({ title, description, timestamp, origin, destination, coords, email, notification: on })
       };
       fetch('/api/events', init)
         .then(() => {
           setTitle('');
           setDescription('');
           setTime('');
           setDate('');
+          setEmail('');
           setOrigin('');
           setDestination('');
           setOriginCoords(null);
           setDestinationCoords(null);
           setTitleError(false);
+          setEmailError(false);
+          setOn(false);
           setDescriptionError(false);
```

Comment:
These error and value functions are starting to get out of control. Try to refactor your code so that you have one object for your form values. That way you only need one change handler.

```js
const [values, setValues] = useState({ 
  title: '',
  description: '',
  // ...
})

setValues({ ...values, [event.target.name]: event.target.value })
```